lib/lvol/lvol.c

```diff
@@ -1378,6 +1433,124 @@ spdk_lvol_create_snapshot(struct spdk_lvol *origlvol, const char *snapshot_name,
 				lvol_create_cb, req);
 }
 
+struct xattrs_ctx {
+	struct spdk_lvol *newlvol;
+	const char *const *xattrs_external;
+};
+
+static void
+lvol_get_xattr_value_ext(void *xattr_ctx, const char *name,
```

Comment:
Extended, because this function extends the function `lvol_get_xattr_value` managing external xattrs too.
Something similar to `spdk_bs_open_blob_ext`, which extends `spdk_bs_open_blob`.